tests/river/e2e/api/test_preview.py

```diff
@@ -1,36 +1,96 @@
+from unittest.mock import patch
+
 import pytest
 
 from django.urls import reverse
 
 from syrupy.filters import paths
 
-pytestmark = pytest.mark.django_db
+pytestmark = [pytest.mark.django_db, pytest.mark.fhir_api]
 
 
 @pytest.mark.export_data("mimic_mapping.json")
-def test_preview(api_client, export_data, snapshot):
-    url = reverse("sources-list")
+class TestPreview:
```

Comment:
We don't use test classes anywhere else in this repo so I don't think it's a good idea to introduce one here